packages/medusa/src/services/cart.ts

```diff
@@ -514,13 +519,45 @@ class CartService extends TransactionBaseService<CartService> {
     return false
   }
 
+  /**
+   * Check if line item's variant belongs to the cart's sales channel.
+   *
+   * @param cart - the cart for the line item
+   * @param lineItem - the line item being added
+   */
+  protected async validateLineItemSalesChannel_(
```

Comment:
suggestion: We could rename it `validateLineItem` and later it could include more validation? wdyt?

Reply:
and in a case that method would validate multiple things we would probably end up with the same `validateLineItemSalesChannel_` method but it would just be called from `validateLineItem`

Reply:
Yes indeed

Reply:
also since this is a TypeScript file (and we have the `protected` keyword), we don't need the "_"-postfix.